src/components/CreateInstructorPage/index.jsx

```diff
@@ -0,0 +1,169 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Helmet from 'react-helmet';
+
+import { CreateInstructorForm } from './CreateInstructorForm';
+import StatusAlert from '../StatusAlert';
+import LoadingSpinner from '../LoadingSpinner';
+
+class CreateInstructorPage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      startedFetching: false,
+    };
+    this.handleInstructorCreate = this.handleInstructorCreate.bind(this);
+    this.setStartedFetching = this.setStartedFetching.bind(this);
+  }
+
+  componentDidMount() {
+    this.props.fetchInstructorOptions();
+    this.setStartedFetching();
```

Comment:
Is this a hack to avoid setting state in componentDidMount because the linter will yell at you?

Is there a more elegant Reacty way to cover the gap before first fetch? Like set a `finishedFetching` state after we get the result or something?

Reply:
Yes, this is to set state here without getting yelled at. I don't think it's that bad to do (it's what we do in prospectus), but I can try and find alternatives if we want